fe/src/lib/app/machines/explore/results.options.js

```diff
@@ -142,8 +142,9 @@ const updateQueue = ctx => {
 	const datasetId = getDatasetIdOf({project, source, version});
 	const aggsToObject = arrayToObjectWith(agg => {
 		const requestWithDefaults = useDefaults(agg.request);
+		const replacedFieldName = (agg.request.field || 'noField').replace('.', '_');
```

Comment:
possibly we could do the below?

```suggestion
		const replacedFieldName = agg.request.field?.replace('.', '_') || 'noField';
```